CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🧬 As a first step to increasing the performance of our JSON import, VAST now
+  relies on [simdjson](https://github.com/simdjson/simdjson) to parse NDJSON.
+  To use the (yet experimental) feature, use `vast import json|suricata
+  --simdjson`. Special thanks to [@ngrodzitski](https://github.com/ngrodzitski)
+  for all the work on this integration.
+  [#1230](https://github.com/tenzir/vast/pull/1230)
+  [#1246](https://github.com/tenzir/vast/pull/1246)
+
```

Comment:
```suggestion
- 🧬 VAST now relies on [simdjson](https://github.com/simdjson/simdjson) for
  JSON parsing. The substantial gains in throughput shift the bottleneck of
  the ingest path from parsing input to indexing at the node. To use the (yet
  experimental) feature, use `vast import json|suricata --simdjson`.
  [#1230](https://github.com/tenzir/vast/pull/1230)
  [#1246](https://github.com/tenzir/vast/pull/1246)
  [@ngrodzitski](https://github.com/ngrodzitski)
```

We are going to acknowledge this external contribution separately in the README. Let's do this in a different PR.